### DISCUSSION TITLE
Agones Health

### DISCUSSION THREAD

**Discussion by rexlManu** [jeremylvln/Shulker#111] [Ideas]
Currently the health check is missing for the game servers. That would need to be added to the agent. 

For testing reasons I already started to add that to the agent. I'm just running the health call with nextOn on a scheduler every fixed amount of seconds.

If thats fine, i can submit it as a pr

**Comment by jeremylvln** [comment]
> Currently the health check is missing for the game servers. That would need to be added to the agent. 
> 
> For testing reasons I already started to add that to the agent. I'm just running the health call with nextOn on a scheduler every fixed amount of seconds.
> 
> If thats fine, i can submit it as a pr

I have read about Agnones healthcheck, but I don't have found a real use for it yet, as live/readiness probes are already fines. But maybe I'm mistaken. I will take the time to read this section a bit more in the documentation to ensure I'm not missing something!

**Comment by jeremylvln** [comment]
This is implemented as part of #112. Do not hesitate to re-open if it does not fit your needs.
